### PR TITLE
8234174: Change IDEA VCS mapping to Git

### PR DESCRIPTION
**Pull request by AlexanderScherbatiy**
Allow IDEA to correctly detect Git version control system when opening JFX project.

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

## Issue
[JDK-8234174](https://bugs.openjdk.java.net/browse/JDK-8234174): Change IDEA VCS mapping to Git

## Approvers
 * Kevin Rushforth ([kcr](@kevinrushforth) - **Reviewer**)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
@AlexanderScherbatiy this is the part that applies to you:

> If you already are an OpenJDK [Author](https://openjdk.java.net/bylaws#author), [Committer](https://openjdk.java.net/bylaws#committer) or [Reviewer](https://openjdk.java.net/bylaws#reviewer), please click [here](https://bugs.openjdk.java.net/secure/CreateIssue.jspa?pid=11300&issuetype=1) to open a new issue so that we can record that fact. Please use "Add GitHub user AlexanderScherbatiy" as summary for the issue.

Also, please read the [contributing guidelines](https://github.com/openjdk/jfx/blob/master/CONTRIBUTING.md). As with the JDK project, we need a bug ID for every pull request.

**Comment by AlexanderScherbatiy** [comment]
/covered my employer BellSoft has signed the OCA

I have created issue JDK-8234174 Change IDEA VCS mapping to Git

**Comment by tsayao** [comment]
Just one note: Idea can import the project from gradle, so many idea files on the project just gets on the way when committing. Not sure about this file in particular, but I think it would be better to remove IDE specific project files (for IDEs that supports it).

File: .idea/vcs.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="hg4idea" />
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
+</project>
```
Comment (kevinrushforth) [inline .idea/vcs.xml]: Minor: can you restore the newline that was removed? It isn't related to the change, and in general, text files should end with a newline (although that's a suggestion, not a rule).

**Comment by AlexanderScherbatiy** [comment]
I use IDEA 2019.2.4.
I tried to remove the .idea directory from jfx project and and open jfx project by IDEA using File -> New -> Project From Existing Sources...  and point to jfx/build.gradle file.
It started to analyze the project and just silently stops importing the project without any message in the ~/.IdeaIC2019.2/system/log/idea.log file.

**Comment by AlexanderScherbatiy** [comment]
Some .idea files are already in [.gitignore](https://github.com/openjdk/jfx/blob/master/.gitignore)
```text
# Ignore IntelliJ files
.idea/tasks.xml
.idea/workspace.xml
.idea/inspectionProfiles/**
.idea/copyright/profiles_settings.xml
.idea/codeStyles/**
.idea/checkstyle-idea.xml
```

File: .idea/vcs.xml (hunk 1 above)
Comment (AlexanderScherbatiy) [reply .idea/vcs.xml]: Updated.

**Pull request closed without merging**